src/hwsyncagent/hwstatemgr/components.py

```diff
@@ -1,7 +1,7 @@
 #
 # MIT License
 #
-# (C) Copyright 2020-2022 Hewlett Packard Enterprise Development LP
+# (C) Copyright 2020-2022, 2024 Hewlett Packard Enterprise Development LP
```

Comment:
Should this be 2020-2024? The comma implies that 2023 isn't included, but I'm sure we owned the copyright then, too.

Reply:
Way back when I was told to just append the current year like this, but it makes no difference to me.

Reply:
Although, I also found this: https://hynek.me/til/copyright-years/